### DISCUSSION TITLE
Nested form object?

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#1324] [Q&A] [answered]
I'm trying `useForm`, and having a form: 
```
{
  user: {
    firstname: "",
    lastname: "",
  },
  ...
}
```
I tried 
```
<TextInput
  required
  label="First name"
  {...form.getInputProps('user.firstname')}
/>
```
and it did not work.

Is there any way to handle with a nested form?

**Comment by rtivital** [comment, marked as answer]
No, currently, form does not support nested objects

**Reply by Toshinaki** [reply]
Thanks.
Any plan on supporting nested form object?

**Reply by Toshinaki** [reply]
@rtivital 
Hi. How about the above question?
Or is there any suggestion about dealing with nested form objects?

**Reply by rtivital** [reply]
Currently it is not planned
